src/Tools/dotnet-counters/ConsoleWriter.cs

```diff
@@ -95,15 +97,25 @@ public void Update(string providerName, ICounterPayload payload, bool pauseCmdSe
             }
             string name = payload.GetName();
             string keyName = CounterNameString(providerName, name);
+            string indent = "    ";
```

Comment:
nit: both of these variables can be marked as `const`.

Reply:
ok, thanks for the feedback!

Reply:
Maybe the value below could be `indent.Length` instead of the literal `4`? Or indent could be `new string(' ', IndentLength);`